lib/codecs/DataUrlEncoder.js

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc. All rights reserved.
+ */
+import {Token, Type} from 'cborg';
+import {Base64} from 'js-base64';
+import {CborldEncoder} from './CborldEncoder.js';
+
+// data URL codec for base64 data
+// https://www.rfc-editor.org/rfc/rfc2397
+// https://fetch.spec.whatwg.org/#data-urls
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs
+// data:[<mediatype>][;base64],<data>
+
+// This codec must be able to round-trip the data. The parsing is minimal and
```

Comment:
We'd normally do a long explanatory doc block with full sentences using multiline comment `/* ... */` chars instead.